### DISCUSSION TITLE
Stronge behaviour with airzone thermostat

### DISCUSSION THREAD

**Discussion by hanor-git** [jmcollin78/versatile_thermostat#216] [Q&A] [answered]
Hello,

I've got a stronge behaviour with an airzone thermostat.
I have configured a versatile thermostat and the underlying device is a airzone thermostat.

When I set up at 22° the versatile thermostat,  my airzone thermostat is setup with the same temp.
But after a few minutes, even if the temp of the sensor used to mesure the temp in the room is below 22, the airzone thermostat and the versatile are setup to 18°.
No possibility to heat.
Any idea ?

Regards,

**Comment by jmcollin78** [comment]
Hello @hanor-git ,

You should post your VTherm attributes so that I can see how you have configure it. It is available in Development Toos / states

**Reply by hanor-git** [reply]
Hello @jmcollin78 

Here some screenshot. 

**setup:**

![setup 22-1](https://github.com/jmcollin78/versatile_thermostat/assets/81289160/877b19f0-d53b-4d84-a3fd-8ec517025266)
![setup 22-2](https://github.com/jmcollin78/versatile_thermostat/assets/81289160/10344278-4bec-4f2f-9290-a8eff49c5a82)
![setup 22-3](https://github.com/jmcollin78/versatile_thermostat/assets/81289160/c3a32953-49a6-4e65-932f-b699a8a72cdf)
![setup 22-4](https://github.com/jmcollin78/versatile_thermostat/assets/81289160/dad101fc-2312-4810-9ad2-47818120f2fd)

**Result after a few minutes**

![Result 18-1](https://github.com/jmcollin78/versatile_thermostat/assets/81289160/665313b3-7e5d-4e28-a4d7-0556991a2073)
![Result 18-2](https://github.com/jmcollin78/versatile_thermostat/assets/81289160/b1d19fd4-a76e-4f8d-8d67-f465bd9df822)
![Result 18-3](https://github.com/jmcollin78/versatile_thermostat/assets/81289160/0ae3c284-d98f-4b4b-a098-6a3ed8ceb9b4)
![Result 18-4](https://github.com/jmcollin78/versatile_thermostat/assets/81289160/92c49fdd-82f8-41bf-8f9f-132a0083b48a)

Bonne soirée,

**Comment by jmcollin78** [comment]
I nothing special on the attributes. If the VTherm change "alone" to 18° after you ask to have 22°, it is certainly the climate.zona_01 which is changing alone. Then VTherm follows the underlying change to allow the usage of the TRV directly.

On the screen I see that VTherm is not only changing to 18° but also is stopped (hvac_off). This confirms the fact that your underlying is switching alone to off and VTherm follows it. I guess it is a feature of the airzone which I don't know.

I fear that VTherm will not be usable nor usefull with this underlying climate. Or maybe you will be able to change setting of you airzone TRV ?

**Comment by hanor-git** [comment, marked as answer]
Thanks for the information. 
I find the solution.
It's the "eco-mode" define in the airzone mobile app who is responsible of this behaviour. You don't have the possibility to define it in HA, you have to do it in the app. Eco mode should be off.
All the best, thanks  for the work.
